### DISCUSSION TITLE
[Feature Request] InlineUpdate Operation

### DISCUSSION THREAD

**Discussion by susanu** [Laravel-Backpack/community-forum#1099] [Ideas]
I sometimes need to update certain relationship items directly from the edit page of the entry without the need to leave the page.

I implemented something like this in backpack v5 and works pretty well.
It works similar to InlineCreate since i took most of the code from there.
Is there any chance you see this implemented into v6 or v7?

Cheers.

**Comment by karandatwani92** [comment]
Yes, you can manage related entries in the same form (create, update, delete). You will find the required info in this part of the [docs](https://backpackforlaravel.com/docs/6.x/crud-fields#manage-related-entries-in-the-same-form-create-update-delete).

**Comment by susanu** [comment]
My use case is:
I have a ```BelongsTo``` relationship and i want to create/update or select from existing entries directly from the update page.
I attach some screenshots below.

![image](https://github.com/user-attachments/assets/7933be75-7f3e-4d87-8d46-ff1f9994c9a5)

When clicking the ```Edit``` button, a popup would appear with fields from ```UpdateOperation```, similar to the popup from ```InlineCreate``` operation.

![image](https://github.com/user-attachments/assets/a404e723-8974-48e8-83f6-aca401bd2c17)
